README.md

```diff
@@ -110,8 +104,7 @@ For detailed explanation on how things work, check out [Nuxt.js docs](https://nu
 
 ### Libraries used  
 
-* [zkSync Checkout lib](https://www.npmjs.com/package/zksync-checkout): our open sourced NPM-package
-* [zkSync.js lib](https://www.npmjs.com/package/zksync-checkout): our open sourced NPM-package
-* [zkCheckout link builder](https://checkout.zksync.io/link): UI to create unique permissionless payment link and get paid in tokens
-* [SDK description](https://zksync.io/api/sdk/checkout/): details on how to use zkCheckout SDK
+* [zkSync Checkout lib](https://www.npmjs.com/package/zksync-checkout): matter labs open sourced NPM-package
+* [RIF Rollup Checkout link builder](https://checkout.aggregation.rifcomputing.net/link): UI to create unique permissionless payment link and get paid in tokens
+* [SDK description](https://docs.aggregation.rifcomputing.net/api/sdk/checkout/): details on how to use Checkout SDK
```

Comment:
![image](https://github.com/rsksmart/rif-rollup-dapp-checkout/assets/6055676/e0104ff0-c295-4c83-bb6b-11eb91118157)
these sdk docs that you are talking about... are in the room with us?

Reply:
hahahaha it's going to be haahahah I guess the end point for [rill-rollup-docs](https://github.com/rsksmart/rif-rollup-docs) will end up in that url, or not?